tests/models/tvlt/test_modeling_tvlt.py

```diff
@@ -0,0 +1,658 @@
+
+    def test_inference_for_question_answering(self):
+        model = TvltForQuestionAnswering.from_pretrained("TVLT/tvlt-base").to(torch_device)
+
+        image_processor, audio_feature_extractor = self.default_feature_extractor
+        video = prepare_video()
+        audio = prepare_audio()
+        video_inputs = image_processor(video, return_tensors="pt").to(torch_device)
+        audio_inputs = audio_feature_extractor(audio, return_tensors="pt").to(torch_device)
+        inputs = dict()
+        inputs.update(video_inputs)
+        inputs.update(audio_inputs)
+        # forward pass
+        with torch.no_grad():
+            outputs = model(**inputs)
+
+        # verify the logits
+        expected_shape = torch.Size((1, 1))
+        self.assertEqual(outputs.logits.shape, expected_shape)
+
+    def test_inference_for_audiovisual_classification(self):
+        model = TvltForAudioVisualClassification.from_pretrained("TVLT/tvlt-base").to(torch_device)
+
+        image_processor, audio_feature_extractor = self.default_feature_extractor
+        video = prepare_video()
+        audio = prepare_audio()
+        video_inputs = image_processor(video, return_tensors="pt").to(torch_device)
+        audio_inputs = audio_feature_extractor(audio, return_tensors="pt").to(torch_device)
+        inputs = dict()
+        inputs.update(video_inputs)
+        inputs.update(audio_inputs)
+
+        # forward pass
+        with torch.no_grad():
+            outputs = model(**inputs)
+
+        # verify the logits
+        expected_shape = torch.Size((1, 1))
+        self.assertEqual(outputs.logits.shape, expected_shape)
+
+    def test_inference_for_pretraining(self):
+        model = TvltForPreTraining.from_pretrained("TVLT/tvlt-base").to(torch_device)
+
+        image_processor, audio_feature_extractor = self.default_feature_extractor
+        video = prepare_video()
+        video_mixed = prepare_video()
+        audio = prepare_audio()
+        video_inputs = image_processor(video, return_tensors="pt", mask_pixel=True).to(torch_device)
+        video_mixed_inputs = image_processor(video_mixed, is_mixed=True, return_tensors="pt").to(torch_device)
+        audio_inputs = audio_feature_extractor(audio, return_tensors="pt", mask_audio=True).to(torch_device)
+        labels = torch.tensor([[0.0]], device=torch_device)
+        inputs = dict()
+        inputs.update(video_inputs)
+        inputs.update(video_mixed_inputs)
+        inputs.update(audio_inputs)
+        inputs.update({"labels": labels})
+
+        # forward pass
+        with torch.no_grad():
+            outputs = model(**inputs)
+
+        # verify the logits
+        expected_pixel_logits_shape = torch.Size([1, 1568, 768])
+        expected_audio_logits_shape = torch.Size([1, 96, 256])
+        expected_matching_logits = torch.tensor([[-1.0838]], device=torch_device)
+        print(outputs.matching_logits)
```

Comment:
Can be removed

Also if possible, would be great to test more than just one logit value